tests/ci/cdk/cdk/codebuild/github_ci_integration_omnibus.yaml

```diff
@@ -0,0 +1,31 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR ISC
+
+version: 0.2
+
```

Comment:
I know it's outside the scope of the story, but can you add in libcurl as an option here?  I'm fine creating a follow-up task but we should have coverage for it since we officially support libcurl now